src/internal/metrics/index.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
In index.ts we inject PACKAGE_VERSION and THEME to metrics utils. This allows all code in internal/metrics except for index.ts to be shared via the component-toolkit.